src/troute-network/troute/NHDNetwork.py

```diff
@@ -1,4 +1,4 @@
-from  .AbstractNetwork import AbstractNetwork
+ from  .AbstractNetwork import AbstractNetwork
```

Comment:
This extra space at the beginning of the line may cause some issues...either way, for consistency it should be removed.